theme.js

```diff
@@ -32,7 +34,8 @@ const theme = {
     tornadoRed: "#ea4335",
     govukYellow: "#FFBF47",
     white: "#ffffff",
-    cardGrey: "#f3f3f3"
+    cardGrey: "#f3f3f3",
+    black: "#000000"
```

Comment:
Are you using black in this PR or was it just since its a good thing to have?

Reply:
I think I was using it at some point but no longer. But it's probably good to have in there.